feat: Add custom dataframe utils to `testcommon` (#71)
Energinet-DataHub/opengeh-python-packages: source/testcommon/release-notes/release-notes.md

# Testcommon Release Notes

## Version 0.0.4

Support custom function `assert_dataframes_and_schemas` for more configuration options.

> As a sideeffect the heavy dependencies on pandas and more can be removed. Greatly
> improving performance of rebuilding docker images depending on this package.

Author: ```suggestion
As a sideeffect the heavy dependencies on pandas and more have been removed. This
greatly improves performance of building docker images depending on this package.
```
